.github/workflows/ci.yml

```diff
@@ -95,3 +95,44 @@ jobs:
         with:
           fail_on_unmatched_files: true
           files: helium-gateway-*.tar.gz
+
+  docker_buildx:
```

Comment:
I'm thinking it may make most sense to moving the git-tag-release triggered build, wich published to quay.io/repository/team-helium/miner, to the release job. And leave the rest as is to publish every build to quay.io/repository/team-helium/test-images